addons/miscFixes/patchHAFMHelis/config.cpp

```diff
@@ -0,0 +1,354 @@
+#include "\z\potato\addons\miscFixes\script_component.hpp"
+#undef COMPONENT
+#define COMPONENT miscFixes_patchHAFMHelis
+
+class CfgPatches {
+    class ADDON {
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = { "potato_core", "HAFM_NH90_Config", "HAFM_NH90", "Kimi_UI", "HAFM_EC635_Config" };
+        skipWhenMissingDependencies = 1;
+        author = "Bourbon Warfare";
+        authorUrl = "https://github.com/BourbonWarfare/POTATO";
+        VERSION_CONFIG;
+    };
+};
+
+class CfgVehicles {
+    class HelicopterWreck;
+    class HAFM_NH90Wreck: HelicopterWreck { XEH_ENABLED; }; // "Wreck" @HAFM
+    class HAFM_EC635Wreck: HelicopterWreck { XEH_ENABLED; }; // "Wreck" 909320014
+
+    class Helicopter_Base_F;
+    class Helicopter_Base_H: Helicopter_Base_F {
+        class UserActions;
+    };
+    class NH90: Helicopter_Base_H {
+        ace_fastroping_enabled = 2;
+        ace_fastroping_friesType = "ACE_friesAnchorBar";
+        ace_fastroping_friesAttachmentPoint[] = {0.02,1.38,-0.37};
+        ace_fastroping_onCut = "ace_fastroping_fnc_onCutCommon";
+        ace_fastroping_onPrepare = "ace_fastroping_fnc_onPrepareCommon";
+        ace_fastroping_ropeOrigins[] = {"ropeOriginLeft","ropeOriginRight"};
+        class textureSources { // enables Virtual Garage skin functionality
+            class NATO {
+                displayName = "NATO (Black)";
+                textures[] = {"HAFM_NH90\data\outerHaul_co.paa","HAFM_NH90\data\upperTail_co.paa","HAFM_NH90\data\misc_co.paa","HAFM_NH90\data\doors_co.paa"};
+            };
+            class AAF {
+                displayName = "AAF";
+                textures[] = {"HAFM_NH90\aaf\AAFouterHaul_co.paa","HAFM_NH90\aaf\AAFupperTail_co.paa","HAFM_NH90\aaf\AAFmisc_co.paa","HAFM_NH90\aaf\AAFdoors_co.paa"};
+            };
+            class GR {
+                displayName = "Hellenic Armed Forces";
+                textures[] = {"HAFM_NH90\gr\GRouterHaul_co.paa","HAFM_NH90\gr\GRupperTail_co.paa","HAFM_NH90\gr\GRmisc_co.paa","HAFM_NH90\gr\GRdoors_co.paa"};
+            };
+            class FIN { // adds Finnish NH90 skin authored by troska and Centurion_101 via SFP: Finnish Forces Pack (https://steamcommunity.com/sharedfiles/filedetails/?id=917042703)
+                displayName = "Finnish Defense Forces";
+                textures[] = {QPATHTOEF(miscFixes,patchHAFMHelis\data\outerHaul_co.paa),QPATHTOEF(miscFixes,patchHAFMHelis\data\upperTail_co.paa),QPATHTOEF(miscFixes,patchHAFMHelis\data\misc_co.paa),QPATHTOEF(miscFixes,patchHAFMHelis\data\doors_co.paa)};
```

Comment:
the actual path is something like `\z\potato\addons\miscFixes\patchHAFMHelis\data\outerHaul_co.paa`
files in these subconfig becomes a real mess, even more so because we also do
`#define COMPONENT miscFixes_patchHAFMHelis`